### PR TITLE
Update browser comparison pages [fix #9378]

### LINKED ISSUE #9378: Update compare hub and tables
## Description

Tyler will provide copy and comparison page changes which will be attached to this bug.

---

#### Success Criteria
- [ ] Changes are specified in a doc attached to this issue, reviewed by The Studio
- [ ] Changes are made in live pages (compare pages + download page)

### PR DESCRIPTION
**Pull request by craigcook**
## Description
Updates browser comparison pages with some copy changes and a few minor corrections.

## Issue / Bugzilla link
#9378 

## Testing
http://localhost:8000/firefox/browsers/compare/

### CODE REVIEW

**Comment by craigcook** [comment]
@peiying2 Can you look over the string changes here?

**Comment by craigcook** [comment]
Marking Do Not Merge for the moment to give Tyler a chance to review.

File: l10n/en/firefox/browsers/compare/safari.ftl
Comment (peiying2) [inline l10n/en/firefox/browsers/compare/safari.ftl:75]: Do we need to change the ID as v2? I know the change is in the the placeable.

Comment (craigcook) [inline l10n/en/firefox/browsers/compare/safari.ftl:75]: This is a bug fix I never noticed until making this update, changing `-brand-name-macos` (which doesn't exist in brands.ftl) to `-brand-name-mac` (which does). I don't think fixing that reference actually makes this a new string... but if it does I can turn this into a new string and mark the old one as obsolete.

Comment (craigcook) [reply l10n/en/firefox/browsers/compare/safari.ftl:75]: Yeah, looks like it does make the whole string new because the placeable doesn't appear in the translations. I'll fix it.

image

**Review by peiying2 — APPROVED** [review]
The .ftl files look good. (It fixed brand placeable missed in the initial round. Thanks!)

**Comment by craigcook** [comment]
Updates have been reviewed by Tyler and her blessing.

**Review by maxxcrawford — APPROVED** [review]
🌎

**Pull request merged**
